### DISCUSSION TITLE
How do you turn off logging?

### DISCUSSION THREAD

**Discussion by soohyunchoi** [griptape-ai/griptape#1119] [Q&A] [answered]
I cannot for the life of me figure out how to turn off logging.

**Comment by collindutter** [comment, marked as answer]
Hey @soohyunchoi, you can disable logging by accessing Griptape's logger and then changing the log level. You can see an example of this [here](https://docs.griptape.ai/stable/griptape-framework/structures/configs/#logging-config).
